api/taxonomies/serializers.py

```diff
@@ -19,20 +19,29 @@ class TaxonomySerializer(JSONAPISerializer):
     filterable_fields = frozenset([
         'text',
         'parents',
+        'parent',
         'id'
     ])
     id = ser.CharField(source='_id', required=True)
     text = ser.CharField(max_length=200)
-    parents = JSONAPIListField(child=TaxonomyField())
+    parents = ser.SerializerMethodField()
```

Comment:
We can deprecate and version it, but it has to continue working the same as it did in earlier versions of the API. There are examples of deprecating fields in the API; I think probably node_links is a good example.